### ISSUE TITLE
Topological sort of the dependency graph

### ISSUE THREAD

**Issue by vijeshm** [auchenberg/dependo#18]
A top sort (a tree-like layout) of the dependency graph would be helpful. This enhancement would help users understand how the modules are structured. 

I say tree-like because,
1. the modules with no dependencies will form the root(s). 
2. cycles may be formed as well.

**Comment by auchenberg** [comment]
Yes. I've been thinking about adding this myself. However I don't know if it's worth spending more time on dependo, now that the [webpack project](http://webpack.github.io/) has built a better analysis tool, http://webpack.github.io/analyse/#warnings
